src/utils/dateTime.ts

```diff
@@ -24,7 +24,7 @@ export const getDateTimeString = (
   date: string, // iso string with offset
   option: "date" | "time"
 ): string => {
-  const zone = DateTime.local().zoneName;
+  const zone = DateTime.local().zoneName ?? DateTime.utc().zone;
```

Comment:
ooo why this

Reply:
Zone name is either string or undefined, ideally, it won't be undefined, but if due to some browser issues if it does come undefined, maybe we can assume they are in UTC?